pkg/replication/replication.go

```diff
@@ -378,6 +407,8 @@ func (r *replication) ensureHealthy(
 		select {
 		case <-quitCh:
 			return
+		case <-r.replicationCancelledCh:
```

Comment:
why is this necessary? it looks like the deferred func in `handleRenewalErrors` will close the `quitCh`